packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/pollingHelper.ts

```diff
@@ -6,7 +6,7 @@ import { AbortSignalLike } from "@azure/abort-controller";
 import {
   CancelOnProgress,
   CreateHttpPollerOptions,
-  LongRunningOperation,
+  RunningOperation,
```

Comment:
is this really expected ? wonder why?

Reply:
Yes there is a renaming in latest preview(proposed by Deya) and see comments https://github.com/Azure/azure-sdk-for-js/pull/29311#discussion_r1575534378

Reply:
@joheredi Could you also share your opionins here? Any idea on a better name?